state/machine_ports_ops.go

```diff
@@ -126,16 +131,38 @@ func (op *openClosePortsOperation) Done(err error) error {
 	return nil
 }
 
-func (op *openClosePortsOperation) verifySubnetAliveWhenSet() error {
-	if op.p.doc.SubnetID == "" {
+func verifySubnetAlive(st *State, subnetID string) error {
+	if subnetID == "" {
```

Comment:
This will be removed in one of the following PRs when subnets will be made mandatory